main/src/main/scala/sbt/internal/BackgroundRunTask.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt
+package internal
+
+import BasicCommandStrings.CompleteExec
+import sbt.internal.util.{ Prompt, Terminal, Util }
+import sbt.util.Logger
+import scala.util.Try
+
+private[sbt] object BackgroundRunTask {
```

Comment:
I am trying to think of a better name for this. `OnAfterJobHandler`? `BackgroundRunTask` itself runs somewhere else, but this does the cleanup / UI handling to emulate as if it ran normally correct?